test/templates/unit_test.tpl

```diff
@@ -17,6 +17,9 @@ jobs:
     - name: Tune the OS
       run: |
         echo '1024 65535' | sudo tee -a /proc/sys/net/ipv4/ip_local_port_range
+        # Increase the asynchronous non-blocking I/O. More information at https://dev.mysql.com/doc/refman/5.7/en/innodb-parameters.html#sysvar_innodb_use_native_aio
+        echo "\nfs.aio-max-nr = 1048576" >> /etc/sysctl.conf
+        sysctl -p /etc/sysctl.conf
```

Comment:
IMO we should simply set the value in the running kernel. IF the same machine is used after rebooting then we’d have this line in the file N times. Any reason not to simply do this:
```
sysctl -w nfs.aio-max-nr=1048576
```

This sets it in the kernel and is idempotent. The file is nice for permanent changes in machines you manage (in our case we have to setup the execution environment each time right?).

Reply:
Yes, I was looking for something like this! All the CI's anyways failed because we can't touch the `etc/sysctl.conf` file without root privilege

Reply:
No this doesn't work, `nfs.aio-max-nr` isn't a configuration that can be changed directly via sysctl